package.json

```
@@ -1,54 +1,53 @@
 {
   "name": "familie-ba-minside-mikrofrontend",
-  "version": "1.0.0",
   "type": "module",
+  "version": "0.0.1",
   "scripts": {
-    "dev": "vite --port 3000",
-    "build": "tsc && vite build",
-    "preview": "vite preview",
-    "test": "vitest",
+    "dev": "astro dev",
+    "start": "astro dev",
+    "build": "astro check && astro build",
+    "mock": "tsx mock/server.ts",
+    "preview": "astro preview",
+    "astro": "astro",
     "prepare": "husky",
-    "lint": "eslint --fix",
-    "typecheck": "tsc --noEmit"
+    "typecheck": "tsc --noEmit",
+    "test": ""
   },
   "dependencies": {
-    "@navikt/aksel-icons": "^7.29.0",
-    "@navikt/ds-css": "5.7.3",
-    "@navikt/ds-react": "5.7.3",
-    "@navikt/nav-dekoratoren-moduler": "3.4.0",
+    "@astrojs/check": "0.9.4",
+    "@astrojs/node": "9.0.0",
+    "@astrojs/react": "4.4.0",
+    "@navikt/aksel-icons": "6.1.1",
+    "@navikt/ds-css": "6.16.2",
+    "@navikt/ds-react": "6.16.2",
+    "@navikt/nav-dekoratoren-moduler": "3.2.3",
+    "@navikt/oasis": "3.4.0",
+    "@opentelemetry/sdk-node": "0.202.0",
+    "astro": "5.14.7",
+    "pino": "9.7.0",
     "react": "18.2.0",
-    "react-dom": "18.2.0"
+    "react-dom": "18.2.0",
+    "rollup-plugin-import-map": "3.0.0"
   },
   "devDependencies": {
-    "@eslint/js": "^9.35.0",
-    "@rollup/plugin-terser": "0.4.4",
-    "@testing-library/react": "14.1.2",
-    "@testing-library/user-event": "14.6.1",
-    "@types/react": "18.2.47",
-    "@types/react-dom": "18.2.18",
-    "@types/react-modal": "3.16.3",
-    "@vitejs/plugin-react": "4.7.0",
-    "eslint": "9.35.0",
-    "eslint-config-prettier": "10.1.5",
-    "eslint-plugin-jsx-a11y": "6.10.2",
-    "eslint-plugin-react": "7.37.5",
+    "@biomejs/biome": "2.2.3",
+    "@hono/node-server": "1.11.2",
+    "@types/node": "20.2.5",
+    "@types/react": "18.2.64",
+    "@types/react-dom": "18.2.21",
+    "hono": "4.6.5",
     "husky": "9.1.7",
-    "jsdom": "23.2.0",
-    "lint-staged": "16.1.2",
-    "prettier": "3.6.1",
-    "rollup-plugin-import-map": "3.0.0",
+    "lint-staged": "16.1.6",
+    "postcss": "8.5.3",
+    "postcss-prefix-selector": "2.1.1",
+    "tsx": "4.7.1",
     "typescript": "5.3.3",
-    "typescript-eslint": "^8.43.0",
-    "vite": "6.3.5",
-    "vite-plugin-css-injected-by-js": "3.5.2",
     "vitest": "3.2.4"
   },
-  "prettier": {
-    "printWidth": 120
-  },
   "lint-staged": {
-    "*.{ts,tsx,css,md,html,json}": [
-      "prettier --cache --write"
-    ]
+    "*.{js,ts,cjs,mjs,d.cts,d.mts,jsx,tsx,json,jsonc}": ["biome check --write --no-errors-on-unmatched"]
```

```suggestion
    "*.{js,ts,cjs,mjs,d.cts,d.mts,jsx,tsx,json,jsonc,astro}": ["biome check --write --no-errors-on-unmatched"]
```

Legger til `astro` slik at de blir formatert ved commit.